lib/octokit/client/contents.rb

```diff
@@ -78,17 +76,20 @@ def create_contents(*args)
             file.close
           end
         end
-        raise ArgumentError.new("content or :file option required") if content.nil?
-        options[:content] = Base64.respond_to?(:strict_encode64) ?
-          Base64.strict_encode64(content) :
-          Base64.encode64(content).delete("\n") # Ruby 1.9.2
+        raise ArgumentError, "content or :file option required" if content.nil?
+
+        options[:content] = if Base64.respond_to?(:strict_encode64)
+  Base64.strict_encode64(content)
```

Comment:
It also looks like this could be removed entirely as it's targeting Ruby 1.9.2. That means that we can merge this as is, and come back in another PR to cut out this condition.

Reply:
Issue created: https://github.com/octokit/octokit.rb/issues/1433